### PR TITLE
Fix twilio numbers

### PR DESCRIPTION
**Pull request by mrichards42**
* Use our messaging service by default, instead of the single long code in `TWILIO_NUMBERS`. Since all of our notys already set a `:from` key, `TWILIO_NUMBERS` isn't likely to be used, so this is more of a just in case change.
* Use real credentials in development, so you can actually send texts. The previously configured credentials were [twilio test credentials](https://www.twilio.com/docs/iam/test-credentials), which afaict are supposed to be used for things like development and automated testing, and don't let you actually send messages. Since we're more or less using the development cluster for staging right now, it seems like we should be able to send messages from it.

### CODE REVIEW

File: deploy/development/sms-works.secret.sops.yml [hunk 1]
```diff
@@ -4,16 +4,18 @@ metadata:
...
+    TWILIO_NUMBERS: ENC[AES256_GCM,data:U3THVCCLpoYwDG7eQTGFkMMZemY6TasldRZi8vFPGT/rfZw+csTEXiEelukADdW+jm1m8Q==,iv:CMnegymbo4ybziT8ulP0TwHE207seNmkjQItlceYiBk=,tag:6N9r1ulX+h97qxxhjuwwgA==,type:str]
 sops:
     kms:
     -   arn: arn:aws:kms:us-west-2:296810667601:key/8b6118da-8ed6-4fe9-8af2-427bcb53b6f1
         created_at: '2018-10-19T18:05:46Z'
         enc: AQICAHgc4HXSGuC1nxk8B4K7M8Cf+gMWV4CRY/KktcxAjthqbAHYegknj4VtNaJq+9cUtmdvAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMUVRaFNAt+1csPA6fAgEQgDsUjvbfBYn/aFIikdTlqt3wRBcxftjfq9x9ph1oH6lRt14RXtUpOZLx5tYiT6l58Y/MeW14/esjyL6zcg==
     gcp_kms: []
-    lastmodified: '2018-10-19T18:05:46Z'
-    mac: ENC[AES256_GCM,data:CCItnxhMi4lgVryaQg7BqDsw8FMMO5nwvYFTi+FOeLc9u2xl8k3sGPRNhg8TMKdLgYUCXr1QAGZXcsnPIxNhj7GszxHr58lJduMacC+/DNE+6aylVNsmcodYM8Ao5Y5qMlZWrqizuuDdATakaQODPyGGX6agwZusRNFqmxLwzog=,iv:7DofE5l+H9DxSRwHKl4bMSEdgAEIbw6i6Vz4UEih+CU=,tag:+OITqfnJnzVWHHToLndgWQ==,type:str]
+    azure_kv: []
```
Comment (eggsyntax) [inline deploy/development/sms-works.secret.sops.yml]: Where did `azure_kv` come from?

Comment (mrichards42) [reply deploy/development/sms-works.secret.sops.yml]: 🤷‍♂️ sops is completely in control of that file. could be related to version 3.0.0 -> 3.1.1? we seem to have that in a lot of repos.

Comment (gws) [reply deploy/development/sms-works.secret.sops.yml]: Yeah, we don't use GCP but its key management service entry is empty in these sops files at the `gcp_kms` key.

Azure Key Vault support was [added in 3.1.0](https://github.com/mozilla/sops/commit/7e7e54bc8c561492b22c62bba31583f0bac15dd8).

**Comment by mrichards42** [comment]
> Does this mean I need to remove the twilio number from my config when I use bulk-send?

The secrets and configmaps are only used in k8s, so you still need to configure your environment when running locally. But it _would_ be good to use that messaging service for `TWILIO_NUMBERS` instead of a single short or long code.

**Pull request merged**